packages/status/src/class-status.php

```diff
@@ -266,4 +268,31 @@ public function is_staging_site() {
 		 */
 		return apply_filters( 'jetpack_is_staging_site', $is_staging );
 	}
+
+	/**
+	 * Returns the site slug suffix to be used as part of Calypso URLs.
+	 *
+	 * Strips http:// or https:// from a url, replaces forward slash with ::.
+	 *
+	 * @since 9.2.0
+	 *
+	 * @param string $url URL to build the site suffix from.
+	 *
+	 * @return string
+	 */
+	public function get_site_suffix( $url = '' ) {
+		// On WordPress.com, site suffixes are a bit different.
+		if ( method_exists( 'WPCOM_Masterbar', 'get_calypso_site_slug' ) ) {
+			return WPCOM_Masterbar::get_calypso_site_slug( get_current_blog_id() );
+		}
+
+		if ( empty( $url ) ) {
+			$url = \home_url();
+		}
+
+		$url = preg_replace( '/.*?:\/\//i', '', $url );
```

Comment:
Could simplify the regex somewhat, there's no need for the `i` flag when there's no letters in the regex and you can avoid [leaning toothpicks](https://en.wikipedia.org/wiki/Leaning_toothpick_syndrome) by using an alternative delimiter.
```suggestion
		$url = preg_replace( '#.*?://#', '', $url );
```